### PR TITLE
Adds BEEF BROTH and BEEF FIZZ

### PR DESCRIPTION
**Pull request by ToasterBan**
## About The Pull Request

this pull request adds BEEF BROTH avaliable from cargo and its disgusting, why would you drink beef broth? you can also make beef fizz if you are insane enough

## Why It's Good For The Game

its not

## Changelog
:cl: Bluezorua and lordscrubling (thanks for unshitting my code)
add: BEEF BROTH
add: BEEF FIZZ
imageadd: BEEF BROTH
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/recipes/drinks_recipes.dm [hunk 1]
```diff
@@ -840,3 +840,9 @@
 	id = /datum/reagent/consumable/ethanol/beesknees
 	results = list(/datum/reagent/consumable/ethanol/beesknees = 4)
 	required_reagents = list(/datum/reagent/consumable/ethanol/mead = 1, /datum/reagent/consumable/honey = 1, /datum/reagent/consumable/ethanol/whiskey = 1, /datum/reagent/consumable/lemonjuice = 1)
+
+/datum/chemical_reaction/beesknees
```
Comment (lordScrubling) [inline code/modules/food_and_drinks/recipes/drinks_recipes.dm:844]: bro

Comment (ToasterBan) [inline code/modules/food_and_drinks/recipes/drinks_recipes.dm]: i fucked this up badly

Comment (BrumalLake) [inline code/modules/food_and_drinks/recipes/drinks_recipes.dm]:
```suggestion
/datum/chemical_reaction/beeffizz
```

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -2170,6 +2170,48 @@
...
 	crate_name = "grilling fuel kit crate"
 
+/datum/supply_pack/organic/beefbroth
+
+	name = "Beef Broth Bulk Crate"
+	desc = "Noone really wants to order beef broth so were selling it in bulk!"
+	cost = 15000
+	contraband = TRUE
+	crate_type = /obj/structure/closet/crate
+	contains = list(/obj/item/reagent_containers/food/snacks/canned/beefbroth
```
Comment (BrumalLake) [inline code/modules/cargo/packs.dm]: all this needs commas after every line thats not last too

Comment (ToasterBan) [reply code/modules/cargo/packs.dm]: alright I'll try to fix this

File: code/modules/cargo/packs.dm [hunk 3]
```diff
@@ -2170,6 +2170,48 @@
 					)
 	crate_name = "grilling fuel kit crate"
 
+/datum/supply_pack/organic/beefbroth
+
+	name = "Beef Broth Bulk Crate"
+	desc = "Noone really wants to order beef broth so were selling it in bulk!"
+	cost = 15000
+	contraband = TRUE
```
Comment (super12pl) [inline code/modules/cargo/packs.dm:2178]: Why is beef broth illegal? :flushed:

Comment (ToasterBan) [reply code/modules/cargo/packs.dm:2178]: why would you like to know?

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 4]
```diff
@@ -876,3 +876,17 @@
 	glass_icon_state = "glass_yellow"
 	glass_name = "glass of bungo juice"
 	glass_desc = "Exotic! You feel like you are on vacation already."
+
+/datum/reagent/consumable/beefbroth
+	name = "Beef Broth"
+	color = "color = "#100800" // rgb: 16, 8, 0 , just like cola
```
Comment (BrumalLake) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]:
```suggestion
	color = "#100800" // rgb: 16, 8, 0 , just like cola
```

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 5]
```diff
@@ -2342,3 +2342,18 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	description = "this is beef fizz, BEEF FIZZ, THERE IS NO GOD"
+	boozepwr = 15
+	quality = DRINK_BAD
+	taste_description = "Nice and Salty Fizzless Beef Juice with a quick bite of lemon"
+	glass_icon_state = "beef_fizz"
+	glass_name = "Beef Fizz"
+	glass_desc = "WHO THOUGHT THIS WAS A GOOD IDEA??"
+
+/datum/reagent/consumable/ethanol/beeffizz/on_mob_metabolize(mob/living/M)
+		to_chat(M, "<span class='warning'>That drink was horrible! You feel sick.</span>")
```
Comment (BrumalLake) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]:
```suggestion
	to_chat(M, "<span class='warning'>That drink was horrible! You feel sick.</span>")
```

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 6]
```diff
@@ -2342,3 +2342,18 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+		M.adjust_disgust(50)
```
Comment (BrumalLake) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]:
```suggestion
	M.adjust_disgust(50)
```

**Review by BrumalLake — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 7]
```diff
@@ -876,3 +876,17 @@
...
+	color = "#100800" // rgb: 16, 8, 0 , just like cola
+	taste_description = "Pure Beef Essence"
+	glass_icon_state  = "glass_brown"
+	glass_name = "glass of Space Cola?"
+	glass_desc = "A glass of what appears to be refreshing Space Cola."
+
+/datum/reagent/consumable/beefbroth/on_mob_metabolize(mob/living/M)
+		to_chat(M, "<span class='warning'>That drink was way too meaty! You feel sick.</span>")
+		M.adjust_disgust(30)
+		SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "quality_drink", /datum/mood_event/quality_bad)
```
Comment (BrumalLake) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: 3 more lines with piss poor indents

Comment (ToasterBan) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm]: my coding skills are amazing aren't they

File: code/modules/food_and_drinks/recipes/drinks_recipes.dm [hunk 8]
```diff
@@ -840,3 +840,9 @@
 	id = /datum/reagent/consumable/ethanol/beesknees
 	results = list(/datum/reagent/consumable/ethanol/beesknees = 4)
 	required_reagents = list(/datum/reagent/consumable/ethanol/mead = 1, /datum/reagent/consumable/honey = 1, /datum/reagent/consumable/ethanol/whiskey = 1, /datum/reagent/consumable/lemonjuice = 1)
+
+/datum/chemical_reaction/beeffizz
+	name = "Beef Fizz"
+	id = /datum/reagents/consumable/ethanol/beeffizz
```
Comment (BrumalLake) [inline code/modules/food_and_drinks/recipes/drinks_recipes.dm]:
```suggestion
	id = /datum/reagent/consumable/ethanol/beeffizz
```

File: code/modules/food_and_drinks/recipes/drinks_recipes.dm [hunk 9]
```diff
@@ -840,3 +840,9 @@
...
+	results = list(/datum/reagent/consumable/ethanol/beeffizz = 10)
+	required_reagents = list(/datum/reagents/consumable/beefbroth = 7, /datum/reagent/consumable/ice = 2, /datum/reagent/consumable/lemonjuice = 1 )
```
Comment (BrumalLake) [inline code/modules/food_and_drinks/recipes/drinks_recipes.dm]:
```suggestion
	required_reagents = list(/datum/reagent/consumable/beefbroth = 7, /datum/reagent/consumable/ice = 2, /datum/reagent/consumable/lemonjuice = 1 )
```

**Review by BrumalLake — CHANGES_REQUESTED** [review]

File: code/game/objects/items/trash.dm [hunk 10]
```diff
@@ -86,6 +86,10 @@
 	name = "Maintenance Peaches"
 	icon_state = "peachcanmaint_empty"
 
+/obj/item/trash/can/food/beefbroth
+	name = "canned beef broth"
+	icon = 'icons/obj/food/food.dmi'
+	icon_state = "beefcan_empty"
```
Comment (Archanial) [inline code/game/objects/items/trash.dm:92]:
```suggestion
	icon_state = "beefcan_empty"

```

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 11]
```diff
@@ -2342,3 +2342,18 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		M.adjustFireLoss(-1.5, 0)
 		M.adjustToxLoss(-1, 0)
 	. = ..()
+
+/datum/reagent/consumable/ethanol/beeffizz
+	name = "Beef Fizz"
+	description = "this is beef fizz, BEEF FIZZ, THERE IS NO GOD"
```
Comment (Archanial) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]:
```suggestion
	description = "This is beef fizz, BEEF FIZZ, THERE IS NO GOD"
```

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 12]
```diff
@@ -876,3 +876,17 @@
...
+	color = "#100800" // rgb: 16, 8, 0 , just like cola
+	taste_description = "Pure Beef Essence"
+	glass_icon_state  = "glass_brown"
+	glass_name = "glass of Space Cola?"
+	glass_desc = "A glass of what appears to be refreshing Space Cola."
+
+/datum/reagent/consumable/beefbroth/on_mob_metabolize(mob/living/M)
+	to_chat(M, "<span class='warning'>That drink was way too meaty! You feel sick.</span>")
+	M.adjust_disgust(30)
+	SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "quality_drink", /datum/mood_event/quality_bad)
```
Comment (Archanial) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm:891]: Why is this send here but not on metabolize with the other one?

**Comment by Raven-Industries** [comment]
Holy cow it actually exists.
image

**Comment by eeSPee** [comment]
Haha use beef fizz to powergame obesity!!!

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm (hunk 12 above)
Comment (ToasterBan) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm:891]: I have no idea this is a horrible mess of spaghetti code that I made mainly by copy and pasting shit that I didn't understand at all

**Comment by ToasterBan** [comment]
> Haha use beef fizz to powergame obesity!!!

fernet already exists if you want to powergame obesity lmao

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 13]
```diff
@@ -2342,3 +2342,18 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	quality = DRINK_BAD
+	taste_description = "Nice and Salty Fizzless Beef Juice with a quick bite of lemon"
+	glass_icon_state = "beef_fizz"
+	glass_name = "Beef Fizz"
+	glass_desc = "WHO THOUGHT THIS WAS A GOOD IDEA??"
+
+/datum/reagent/consumable/ethanol/beeffizz/on_mob_metabolize(mob/living/M)
+	to_chat(M, "<span class='warning'>That drink was horrible! You feel sick.</span>")
+	M.adjust_disgust(50)
+	. = ..()
```
Comment (utilbird) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]:
```suggestion
	return ..()
```

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm (hunk 12 above)
Comment (ike709) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm:891]: Add it to the other one as well.

File: code/modules/cargo/packs.dm [hunk 14]
```diff
@@ -2170,6 +2170,48 @@
...
+	crate_type = /obj/structure/closet/crate
+	contains = list(/obj/item/reagent_containers/food/snacks/canned/beefbroth,
```
Comment (ike709) [inline code/modules/cargo/packs.dm:2180]: Tone this down quite a bit, there shouldn't be more than 4-8 at the most.

File: code/modules/cargo/packs.dm [hunk 15]
```diff
@@ -2170,6 +2170,48 @@
 					)
 	crate_name = "grilling fuel kit crate"
 
+/datum/supply_pack/organic/beefbroth
+
+	name = "Beef Broth Bulk Crate"
+	desc = "Noone really wants to order beef broth so were selling it in bulk!"
+	cost = 15000
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	cost = 5000
```

File: code/modules/cargo/packs.dm [hunk 16]
```diff
@@ -2170,6 +2170,48 @@
 					)
 	crate_name = "grilling fuel kit crate"
 
+/datum/supply_pack/organic/beefbroth
+
+	name = "Beef Broth Bulk Crate"
+	desc = "Noone really wants to order beef broth so were selling it in bulk!"
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	desc = "No one really wants to order beef broth so we're selling it in bulk!"
```

File: code/modules/cargo/packs.dm (hunk 14 above)
Comment (ToasterBan) [reply code/modules/cargo/packs.dm:2180]: yes but 32 cans of beef broth is funny

**Review by utilbird — APPROVED** [review]
Beef broth would make a good maint item

**Pull request merged**